addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12711,7 +12711,11 @@ msgctxt "#24087"
 msgid "All repositories"
 msgstr ""
 
-#empty strings id 24088
+#. Used in add-on browser
+#: xbmc/filesystem/AddonsDirectory.cpp
+msgctxt "#24088"
```

Comment:
we might be able to reuse label 593 - at least the context should be similar enough